src/telemetry.ts

```diff
@@ -61,11 +61,18 @@ export class TelemetryOutputChannel implements vscode.OutputChannel {
   }
 
   private checkError(value: string): void {
+    if (!value) {
+      return;
+    }
     if (value.startsWith('[Error') || value.startsWith('  Message: Request')) {
       if (this.isNeedToSkip(value)) {
         return;
       }
-      this.telemetry.send({ name: 'yaml.server.error', properties: { error: this.createErrorMessage(value) } });
+      const errorMessage = this.createErrorMessage(value);
+      if (errorMessage === 'null') {
```

Comment:
why is this even happening?

Reply:
It happened if some code in yaml-ls calls `console.error()` with null parameter, I found only one place where that can happened, going to add more info there.

Reply:
I create https://github.com/redhat-developer/yaml-language-server/pull/506 which, hopefully, provide more info for `null` messages. 

And this PR is not necessary.